_data/teamtexts.yml

```diff
@@ -34,7 +34,6 @@
 
 - htmlsection: community
   title: Community Contributors
-  text: Qubes would not be where it is today without the input of the many users, testers, and developers of all skill levels who have come together to form this thriving community. The community's discussions take place primarily on the
-  mail: Qubes mailing lists
+  text: Qubes would not be where it is today without the input of the many users, testers, and developers of all skill levels who have come together to form this thriving community. The community's discussions take place primarily on the <a href="/support/">Qubes mailing lists and forum</a>
```

Comment:
This is definitely not the intended way to fix this.

I doubt there's supposed to be HTML in this text snippet. If you look here, you can see the problematic line:

https://github.com/QubesOS/qubesos.github.io/blob/160ef082af2757e18411e40a40be67302cea0bf6/_includes/team.html#L173

The problem is that I can't find `support_link` anywhere else. Looks like this was introduced in https://github.com/QubesOS/qubesos.github.io/pull/153. @maiska, what did you have in mind here?

Reply:
You're right, there is probably a better way. This method works (it renders correctly), I don't know enough about jekyll to know a better way.

Reply:
Hello, sorry for my long silence, I should be more flexible from now on. This here is clearly a mistake, sorry for that, this should be declared a variable which slipped the commit if I am not mistaken in the history of the file, such as `{% assign team_link = lang | append: "/team/" %}`
somthing like that:
`{% assign support_link = lang | append: "/support/" %}`

Reply:
> Hello, sorry for my long silence, I should be more flexible from now on. This here is clearly a mistake, sorry for that, this should be declared a variable which slipped the commit if I am not mistaken in the history of the file, such as `{% assign team_link = lang | append: "/team/" %}`
> 
> somthing like that:
> 
> `{% assign support_link = lang | append: "/support/" %}`

@maiska: Take a look at what has already been merged and please submit a new PR if warranted.

Reply:
@andrewdavidwong I see you fixed it :) Thank you